### PR TITLE
Create interactive GraphQL visual tool [Fixes #1243]

### LINKED ISSUE #1243: Create interactive GraphQL visual tool
**Is your feature request related to a problem? Please describe.**
The API GraphQL schema is constantly changing and it is difficult to manually document it for all the developers who need to use it in the following repos:

1. Talawa
2. Talawa-API
3. Talawa-Admin

**Describe the solution you'd like**
We have identified an open source project that can visually represent a GraphQL schema which would be useful to be integrated into the API for all future development.  

Visualizer:

- https://ivangoncharov.github.io/graphql-voyager/
- https://github.com/IvanGoncharov/graphql-voyager 

We would like to:

1. use this software to automatically visualize the latest schema, preferably with optional activation.
2. keep the generator code within the Talawa-API repo in a directory tree reserved for GitHub automation (if advisable)

**Describe alternatives you've considered**

N/A

**Approach to be followed (optional)**

See above

**Additional context**

1. The https://github.com/exogen/graphql-markdown installation is incorrect.
2. The build process creates a src/index.js file which the documentation refers to as the `graphql-markdown` executable.

### PR DESCRIPTION
**Pull request by EshaanAgg**
**What kind of change does this PR introduce?**
Feature

**Issue Number:**

Fixes #1243 

**Did you add tests for your changes?**
NA

**Snapshots/Videos:**
image

**If relevant, did you update the documentation?**
Yes

**Summary**
Introduced a new endpoint to the API, `/voyager`, that can be accessed by the developers to use the visualizer.

**Does this PR introduce a breaking change?**
No

**Other information**
NA

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

File: src/typeDefs/README.md [hunk 1]
```diff
@@ -79,3 +79,11 @@ You can use `#` symbol to comment out stuff in the graphQL schema as shown for t
 # Schema represents a contract
 
 A graphQL schema represents a contract between the server and the clients. Be very careful when editing the schema as you can unknowingly edit stuff which breaks functionality for all clients relying on the schema. Read the [graphQL spec](https://spec.graphql.org/October2021/) for more information.
+
+# GraphQL Voyager
+
```
Comment (palisadoes) [inline src/typeDefs/README.md:84]: Put all of this in CONTRIBUTING.md Nobody is going to find it here.

**Review by palisadoes — CHANGES_REQUESTED** [review]
See requested changes.

File: src/server.ts [hunk 2]
```diff
@@ -211,6 +214,12 @@ const serverStart = async () => {
           apolloServer.subscriptionsPath
         }`
       );
+      logger.info(
+        "\x1b[1m\x1b[32m%s\x1b[0m",
+        `🚀 Visualise the schema at ws://localhost:${
```
[the remark this answers is not in the document]
Comment (EshaanAgg) [reply src/server.ts:219]: Yes. Missed this change.

File: package.json [hunk 3]
```diff
@@ -106,5 +107,10 @@
     "tsx": "^3.11.0",
     "typescript": "^4.8.4",
     "vitest": "^0.24.3"
+  },
+  "overrides": {
+    "graphql-voyager": {
+      "graphql": "^15.8.0"
```
Comment (xoldyckk) [inline package.json:113]: why is this needed?

File: src/server.ts [hunk 4]
```diff
@@ -85,6 +86,8 @@ app.use(
 app.use("/images", express.static(path.join(__dirname, "./../images")));
 app.use(requestContext.middleware());
 
+app.use("/voyager", voyagerMiddleware({ endpointUrl: "/graphql" }));
```
[the remark this answers is not in the document]
Comment (EshaanAgg) [reply src/server.ts:89]: The `/vogayer` consumes the `/graphql` endpoint to run an introspection query to generate the visualizations. 

I'll modify the server to expose the same only in development environment.

File: package.json (hunk 3 above)
Comment (EshaanAgg) [reply package.json:113]: GraphQL-Voyager needs graphql to be `16.8.0` and above by default. We are overwriting the same here so that the users don't have to provide `--force` flag while installing the same.

**Review by xoldyckk — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]
